alf/nest/cnest/setup.py

```diff
@@ -123,7 +123,7 @@ def build_extensions(self):
 
 setup(
     name="cnest",
-    setup_requires=['pybind11==2.5.0'],
+    setup_requires=['pybind11>=2.5.0'],
```

Comment:
For some historical reason in this PR https://github.com/HorizonRobotics/alf/pull/711 , pybind11 was fixed to 2.5.0.

Reply:
Not an expert in Python, but I think maybe a certain version of `2.5.x` is harmful. The current latest version is `2.6.2` and we just need to avoid the bad version.

In my development environment the version is fixed to `2.6.2` and it works well.

Reply:
OK then maybe the requirement here should be >=2.6.2 instead?